Subida de imagenes a AWS

rojas1995 opened the pull request:
Para probarlo se necesitas las keys de amazon

Commit:
[feat] Subir imagenes

Ahora los usuarios pueden subir sus imagenes

Commit:
[fix] Ahora no puedes subir archivos que no sean png o jpg

Commit:
[feat] Fotos del evento

Ahora tambien se pueden subir fotos para un evento

Commit:
[fix] He quitado la key de AWS del Settings

Commit:
[fix] Cambio para AWS, le quito el session token

rojas1995 committed:
Merge branch 'dev' into feat/be/M-10

rojas1995 commented:
El travis peta porque no tiene puestas las keys en el settings

angbongon commented:
> El travis peta porque no tiene puestas las keys en el settings

Vale vale, aún así veo que SonarCloud da un porcentaje de duplicación muy alto, mira si puedes mejoraralo, por lo que he visto es referente a un JavaScript

angbongon commented:
Ah por cierto resuelve los conflictos también

rojas1995 committed:
Merge branch 'dev' into feat/be/M-10

Commit:
[fix] Ahora se puede subir cualquier tipo de imagen

Tambien se ha añadido una restricción para el tamaño de las imagenes, no han de ser superiores a 5mb

Commit:
[refactor] Quito un print que se habia escapado

Commit:
[fix] Elimino form repetido de la imgen en el formulario

rojas1995 committed:
Merge branch 'dev' into feat/be/M-10

Commit:
[fix][refactor] He quitado código duplicado y he puesto algo en el form

En el form he puesto el campo picture que anteriormente había quitado por equivocación

angbongon committed:
Merge branch 'dev' into feat/be/M-10

angbongon commented:
Es posible hacer la subida de imágenes sin utilizar una petición GET, porque veo que en las urls tienes que poner todo lo referente a la imagen. Y por otro lado por qué es necesario usar urls?, no debería, simplemente, utilizarse la url de update/create de profile o event

Commit:
[fix] Añado el decorator del login
